### ISSUE TITLE
Warning that step.id is null when the Wizard is first time rendering.

### ISSUE THREAD

**Issue by wzard** [americanexpress/react-albus#29]
Hi, 

First of all great library. Does what it says and does it best. 
I was trying it out and was getting warning. 
`Warning: Failed prop type: The prop `step.id` is marked as required in `Steps`, but its value is `null`. in Steps (created by Wizard) in Wizard (created by Onboarding)`

Turns out the same warning was replicated in the console.log of the [official examples](http://americanexpress.io/react-albus/) from add animation onwards.

Problem: 
Basically whenever I try to render Step, using Wizard's render api, this warning shows up.
The default props of Steps.id = null. When the object is rendering it throws an error. 

Can you please check?

**Comment by crosscompile** [comment]
Thanks!

I think we probably shouldn't mark `step.id` as required in `<Steps>` since it is purposely `null` for the first render.

This shouldn't cause any issues with functionality but I'll try to open a PR soon to get rid of the warning.

**Comment by wzard** [comment]
Great. Thanks !

**Comment by AntonKL** [comment]
Any update on this one @crosscompile ?

**Comment by dugasmark** [comment]
@AntonKL To prevent the warning to pop I contionnaly set the "step" prop like this:

<Steps key={step.id} step={step.id ? step : undefined}>

This way while the step id is null the "step" props is not pass

**Comment by romarioraffington** [comment]
Seems like a simple fix. @crosscompile why not accept a PR to fix the issue?

**Comment by ryhinchey** [comment]
`onNext` probably shouldn't be called on initial render.  That'd solve for https://github.com/americanexpress/react-albus/issues/29 and https://github.com/americanexpress/react-albus/issues/33.
